pkg/volume/emptydir/quota.go

```diff
@@ -84,6 +94,11 @@ func (cr *realQuotaCommandRunner) RunApplyQuotaCommand(fsDevice string, quota re
 	return "", stderr.String(), err
 }
 
+func (cr *realQuotaCommandRunner) RunMountOptionsCommand() (string, string, error) {
```

Comment:
@pmorie 
![llama](https://cloud.githubusercontent.com/assets/51265/14293949/2db32216-fb45-11e5-8abd-b65551a23f62.gif)

I think I did that to be consistent with the other commands, I might have been pondering making them return a struct at some point, but it looks like 3 of the 4 have no need to return stderr. I will remove it from all three.